Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -223,7 +235,8 @@
         "UnabletofindthelocationofMSDeployfromregistryonmachineError": "Unable to find the location of MS Deploy from registry on machine (Error : %s)",
         "Nopackagefoundwithspecifiedpattern": "No package found with specified pattern",
         "MorethanonepackagematchedwithspecifiedpatternPleaserestrainthesearchpatern": "More than one package matched with specified pattern. Please restrain the search patern.",
-        "Trytodeploywebappagainwithappofflineoptionselected": "Try to deploy web app again with app_offline option selected.",
+        "Trytodeploywebappagainwithappofflineoptionselected": "Try to deploy app service again with app_offline option selected.",
```

Comment:
Since you renamed app_offline, can you make this consistent with the label name you introduced.